evap/staff/templates/staff_text_answer_warnings.html

```diff
@@ -0,0 +1,141 @@
+                                {% for hidden in form.hidden_fields %}
+                                    {{ hidden }}
+                                {% endfor %}
+                                <td class="movable">
+                                    <span class="movable-icon fas fa-arrows-alt-v"></span>
+                                </td>
+                                <td>
+                                    {% include 'bootstrap_form_field_widget.html' with field=form.trigger_strings %}
+                                </td>
+                                <td>
+                                    {% include 'bootstrap_form_field_widget.html' with field=form.warning_text_de %}
+                                </td>
+                                <td>
+                                    {% include 'bootstrap_form_field_widget.html' with field=form.warning_text_en %}
+                                </td>
+                                <td class="align-right">
+                                    {% include 'bootstrap_form_field_widget.html' with class="d-none" field=form.DELETE %}
+                                </td>
+                            </tr>
+                        {% endfor %}
+                    </tbody>
+                </table>
+            </div>
+        </div>
+        <div class="card card-submit-area text-center mb-3">
+            <div class="card-body">
+                <button type="submit" class="btn btn-primary">{% trans 'Save text answer warnings' %}</button>
+            </div>
+        </div>
+        <div class="card">
+            <div class="card-header">
+                {% trans 'Preview' %}
+            </div>
+            <div class="card-body">
+                <p>
+                    {% blocktrans %}
+                        Changes of the form above are only reflected after the form has been saved.
+                    {% endblocktrans %}
+                </p>
+                <div class="row">
+                    <div class="col-question col-lg-4 col-xl-5 d-flex flex-column">
+                        <label for="preview-textarea">
+                            {% trans 'Test textarea' %}
+                        </label>
+                        <span class="mt-2 warning-label d-none" data-warning="meaningless">
+                            <span class="fas fa-exclamation-triangle"></span>
+                            {% trans "This answer will probably be filtered out. You can simply leave the textbox empty." %}
+                        </span>
+                        {% for warning in text_answer_warnings %}
+                            <span class="mt-2 warning-label d-none" data-warning="trigger-string-{{ forloop.counter0 }}">
+                                <span class="fas fa-exclamation-triangle"></span>
+                                {{ warning.warning_text }}
+                            </span>
+                        {% endfor %}
+                    </div>
```

Comment:
This part (obviously) is extemely similar to the element on the vote page. Have you considered factoring it out? It would remove duplication and make sure that the preview always matches the actual form element.

Reply:
I have considered to factor out the entire text answer row, but not the (smaller) warning text elements. Will do it.